app/src/main/res/values-fr/strings.xml

```diff
@@ -129,5 +129,7 @@
     <string name="auto_synchro_disabled">La synchronisation automatique est désactivée</string>
     <string name="enable_auto_synchro_text">Les notifications nécessitent l\'activation de la synchronisation automatique pour fonctionner.\nVoulez-vous ouvrir les paramètres ?</string>
     <string name="open">Ouvrir</string>
+    <string name="back">Retour</string>
```

Comment:
You even add french translation!

Reply:
I added it just for completion. My last French class was 30 years ago, so I would recommend you to double-check my translations!